docs/examples/index.md

```diff
@@ -1,8 +1,134 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: 'Code examples'
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+## Different ways how to pass an input
 
-There is nothing here yet.
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+const input = { hashtags: ['twitter'], resultsLimit: 20 };
```

Comment:
How about choosing a bit different hashtag (it's a bit weird combination no? "Instagram posts for twitter hashtag"